module/CourseInfo.h

```diff
@@ -0,0 +1,57 @@
+/**
+ *@file CourseInfo.h
+ *@brief コース情報
+ *@author higuchi/miyashita
+ */
+
+#ifndef COURSE_INFO_H
+#define COURSE_INFO_H
+
+#include "Range.h"
+
+/**
+ * コース情報を保持するクラス
+ */
+class CourseInfo {
+ public:
+  /**
+   * 受け取った輝度の値を目標値に設定し、区画の名前と距離を初期化する
+   * @param _brightness 目標とする輝度
+   */
+  CourseInfo(int _brightness);
+
+  /**
+   * 目標とする輝度を返す
+   * @return 目標とする輝度
+   */
+  int getBrightness();
```

Comment:
```suggestion
  int getTargetBrightness();
```

オプション (変えても変えなくてもどっちでもいい)
APIにカラーセンサーから輝度を取得するgetBrightness関数があるから、同じ名前だと紛らわしいかも？

Reply:
ついでに、メンバ変数も `brightness` から `targetBrightness` に変更してもよい気がします。

これも、オプションですが...